public/app/return.php

```diff
@@ -0,0 +1,21 @@
+<?php include ('../../includes/layouts/header.php')?>
+        <section class="issue-section">
+            <form action="issue.php" method="post">
+                <div class="form-issue">
+                    <h1>Return Book</h1>
+                    <hr>
+                    <label for="bookNo"><b>Book Number</b></label>
+                    <input type="text" placeholder="Book Number..." name="bookNo" class="input-field" required>
+
+                    <label for="libraryNo"><b>Library Number</b></label>
+                    <input type="text" placeholder="Library Number..." name="libraryNo" class="input-field" required>
+
+                    <label for="datedue"><b>Date Due</b></label>
```

Comment:
you can comment out this date due for now, it will later be used to determine the fine based on how long a book is held after datedue